test/Tests/src/Network/Http_Spec.enso

```diff
@@ -8,6 +8,7 @@ import Standard.Base.Network.HTTP.Request_Body.Request_Body
 import Standard.Base.Network.HTTP.Request_Error
 import Standard.Base.Network.Proxy.Proxy
 import Standard.Base.Runtime.Context
+from Standard.Base.Network.HTTP import resolve_headers
```

Comment:
This test is only executed if:
```
   ## To run this test locally:
       $ sbt 'simple-httpbin/run localhost 8080'
       $ export ENSO_HTTP_TEST_HTTPBIN_URL=http://localhost:8080/
```

Why don't we move the code from `simple-httpbin` into [enso-test-java-helpers](https://github.com/enso-org/enso/tree/d139449bb692a3d122325df6a9220ff021bd71dc/test/Tests/polyglot-sources/enso-test-java-helpers) and let this `Http_Spec` start the server on its own?

Reply:
That sounds like a really cool idea indeed!

Reply:
I'll break this out as a separate task.

Reply:
https://github.com/enso-org/enso/issues/7903